### ISSUE TITLE
Accesibility options not allowed

### ISSUE THREAD

**Issue by gabyardiles** [react-native-community/datetimepicker#187]
# Question
Accessibility Options not allowed

We are using expo 3.20.1 and we want to make some changes but we are not finding how. Some of the things we need are:
* Modify styles such as the background color of the letters when they are not in focus in both date and time,
* When selecting a disabled date that only mentions the voice over that date

is it possible to customize these things?

Thanks, greetings.

**Comment by rodrieja** [comment]
Hi

I have the same issue.

The contrast ratio between text and its background is not at least 4.5:1.
As a low vision/Colorblind user, I should be able to see the text with sufficient contrast with its background so I would be able to read the content.

Current ratio:
For the text “Thu, Apr 23” and "7 AM " (approximate hex:#FFFFFF) on the blue background color(approximate hex:#5383C3) has an approximate insufficient contrast ratio of 3.9:1.

 Expected ratio:
 Regular text (under 18 points regular font or 14 points bold font) must have a contrast ratio of at least 4.5:1 with the background.

![image](https://user-images.githubusercontent.com/8691226/83293788-a1e3a700-a1c2-11ea-9d4f-96a4416faa1b.png)

**Comment by rodrieja** [comment]
Also having problems to navigate using talkback.

As a Talkback user, the location of the elements must be consistent so that he would be able to navigate or use the widget easily.
Current behaviour:
1. focus the '>' (next month) button in the month of april and may.
2. Observe that the 'next >' month button receives the focus after the calendar dates in the month of april and receives the focus before the calendar dates in the month of may, june and other future months.

Expected behaviour:
2. The element must receive the focus consistently either before or after the calendar dates for all the months.
![image](https://user-images.githubusercontent.com/8691226/83293912-d0618200-a1c2-11ea-8718-96f15c89af9d.png)
![image](https://user-images.githubusercontent.com/8691226/83293937-dd7e7100-a1c2-11ea-8b5c-7487b95c8553.png)

**Comment by katka-slivova** [comment]
Hello! I am encountering challenges with accessibility on Android. Specifically, I am unable to adjust the size of the cells in the calendar to meet the minimum requirement of 48dp for accessibility compliance. Currently, the cells are sized at 35dp x 40dp, which is below the necessary threshold. This issue is being flagged by the accessibility scanner.

I am inquiring if there are any plans to address this limitation, as enhancing customization options for Android styling could facilitate meeting these accessibility standards.

Thank you for your attention to this matter.
![2 5 8 Target Size](https://github.com/react-native-datetimepicker/datetimepicker/assets/97403454/537700cd-ed16-43e9-a48f-d1d9dfb30fdf)
